src/averageAges.js

```diff
@@ -15,11 +15,22 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const filteredPeople = people.filter(person => {
```

Comment:
Better use the variable name related with word "men". Because you are using the same name for women.

Reply:
I can't entirely agree with this because the filter function receives all values of people object, not only men's or women's objects and only then it returns men's or women's objects.